maui/src/Charts/Axis/ChartAxis.cs

```diff
@@ -15,7 +15,7 @@ public abstract partial class ChartAxis : Element, IThemeElement
 		double _previousLabelCoefficientValue = -1;
 		CartesianChartArea? _cartesianArea;
 		internal bool _isOverriddenOnCreateLabelsMethod;
-		DoubleRange _actualRange = DoubleRange.Empty;
+		DoubleRange _actualRange = new DoubleRange(double.NaN, double.NaN);
```

Comment:
I'm thinking of something, let me know if that's something worth considering:

what if, in `DoubleRange`'s constructor, we set `start` and `end` values to `double.NaN` by default.

so instead of 
https://github.com/syncfusion/maui-toolkit/blob/f696f6cd725cd6340199461c13bdf5bb07510a64/maui/src/Charts/Utils/DoubleRange.cs#L112

we would have:
```Csharp
 public DoubleRange(double start = double.NaN , double end = double.NaN)
```

So we don't have to repeat `new DoubleRange(double.NaN, double.NaN)` every single time, we could just do `new DoubleRange()`